.github/ISSUE_TEMPLATE/bump_gmt_checklist.md

```diff
@@ -24,6 +24,7 @@ assignees: ''
   - [ ] Update `.github/workflows/ci_docs.yml`
   - [ ] Update `.github/workflows/ci_tests.yaml`
   - [ ] Add the legacy GMT version to `.github/workflows/ci_tests_legacy.yaml`
+  - [ ] Update `extlinks` in `doc/conf.py`
```

Comment:
Make it a bit clearer that the 6.x version is the one to be updated? Feel free to reword.
```suggestion
  - [ ] Update GMT 6.x minor version for `extlinks` urls in `doc/conf.py`
```